### PR TITLE
create a button for instruction

### PR DESCRIPTION
**Pull request by Maichonok**
I created a modal window for instruction

### CODE REVIEW

File: sokoban/src/App.tsx [hunk 1]
```diff
@@ -33,6 +32,7 @@ function App() {
 
   return (
     <>
+    <InstructionButton/>
```
Comment (SofiaHjerpe) [inline sokoban/src/App.tsx:35]: Perhaps put button under or to the right?

File: sokoban/src/components/Instruction.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
+.modal-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
```
Comment (SofiaHjerpe) [inline sokoban/src/components/Instruction.css:8]: Check styles if anything is unused?

File: sokoban/src/components/Instruction.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+const InstructionModal: React.FC<ModalProps> = ({ onClose }) => {
+  return (
+    <div className="modal-overlay">
+      <div className="modal-content">
+        <h2>Instruction</h2>
+        <p>
+          Sokoban (倉庫番, Sōko-ban, lit.'warehouse keeper') is a puzzle
+          video game in which the player pushes boxes around in a warehouse,
```
Comment (SofiaHjerpe) [inline sokoban/src/components/Instruction.tsx:15]: Instruction text needs to be updated (we will look at it together) :)

File: sokoban/src/App.tsx [hunk 4]
```diff
@@ -2,6 +2,7 @@ import { useState } from "react";
 import { GameArray } from "./Globals";
 import { Form } from "./components/Form";
 import "./game.css";
+import InstructionButton from "./components/Instruction";
```
Comment (SofiaHjerpe) [inline sokoban/src/App.tsx:6]: Put all simular imports together, should game.css be under this line?

**Review by Dakawe — APPROVED** [review]

File: sokoban/src/components/Instruction.tsx (hunk 3 above)
Comment (Maichonok) [reply sokoban/src/components/Instruction.tsx:15]: Okay, I can add later :)

**Comment by Maichonok** [comment]
have done 
Screenshot 2024-03-19 at 02 09 44

**Pull request merged**